bin/oneoff/data_fix/time_spent_milliseconds_to_seconds

```diff
@@ -19,18 +19,11 @@ require_relative '../../../dashboard/config/environment'
 # upper limit: 2811329000;
 # lower limit:   93221000;
 
-slice = 0
-
-UserLevel.find_in_batches(start: 93_221_000, finish: 2_811_329_000, batch_size: 5000) do |user_level_slice|
-  puts "PROCESSING: slice #{slice} with starting id #{user_level_slice.first.id}."
-  ActiveRecord::Base.transaction do
-    user_level_slice.each do |user_level|
-      if user_level.time_spent && user_level.time_spent > 0
-        user_level.time_spent = (user_level.time_spent.to_f / 1000).ceil.to_i
-        user_level.save(touch: false, validate: false)
-      end
-    end
+UserLevel.find_each(start: 93_221_000, finish: 2_811_329_000, batch_size: 5000) do |user_level|
```

Comment:
Could we use `find_each.with_index` to track where we are and occasionally (every 5000 rows?) output where we are, so we can monitor this long running process similar to the way we were previously logging the start of each new slice?

Reply:
👍